frb_example/with_flutter/lib/bridge_generated.dart

```diff
@@ -220,6 +221,9 @@ class FlutterRustBridgeExampleImpl implements FlutterRustBridgeExample {
         argNames: [],
       );
 
+  void close() {
```

Comment:
uber-nit: shall we use `dispose` instead of `close`, which is more Dart/Flutter-like